services/private-gke/client-experimentation/gke-autopilot/gke-autopilot.yaml

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# GKE Autopilot Cluster
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+metadata:
+  name: exp-cluster # kpt-set: ${cluster-name}
+  namespace: config-control # kpt-set: ${project-namespace}
+  annotations:
+    cnrm.cloud.google.com/project-id: project-id # kpt-set: ${project-id}
+spec:
+  addonsConfig:
+    configConnectorConfig:
+      enabled: false
+    dnsCacheConfig:
+      enabled: true
+  description: experimentation autopilot gke cluster # kpt-set: ${cluster-description}
+  enableAutopilot: true
+  binaryAuthorization:
+    evaluationMode: PROJECT_SINGLETON_POLICY_ENFORCE
+  enableIntranodeVisibility: true
+  enableShieldedNodes: true
+  initialNodeCount: 1
+  ipAllocationPolicy:
+    # Cluster Secondary Range name and CIDR will be auto-created
+    clusterSecondaryRangeName: ""
```

Comment:
Is there a reason we're not using a named range and relying on the auto-created CIDRs?

Reply:
I believe I had changed to using auto-generated ranges while I was troubleshooting a flow issue with the PSC configuration. I can switch back if necessary. I would not use this config outside of a sandbox environment.

Reply:
@cartyc Fixed! Let me know if I missed anything.